README.md

```diff
@@ -1,2 +1,47 @@
-# bc_access_menu
-This is the App to display the Link back to the ACCESS Portal on the Navigation bar.
+# Batch Connect - Access OnDemand Menu
+
+![GitHub Release](https://img.shields.io/github/release/osc/bc_access_menu.svg)
+[![GitHub License](https://img.shields.io/badge/license-MIT-green.svg)](https://opensource.org/licenses/MIT)
+
+This is a menu item to display a link back to the Access OnDemand Web portal on the Navigation Bar of Open OnDemand.
+
+## Install
+
+Use Git to clone this app and checkout the desired branch/version you want to
+use:
+
+```sh
+git clone <repo>
+cd <dir>
+git checkout <tag/branch>
+```
+
+To activate this menu item on your Open OnDemand Portal, you need to:
+
+```sudo cp -Rf bc_access_menu /var/www/ood/apps/sys/```
```

Comment:
The install steps should probably mirror the steps taken by RPM.  If you copied this repo to that location, would miss the sub-directory apps.